src/Raven.Server/Documents/Indexes/Persistence/Lucene/IndexReadOperation.Sharding.cs

```diff
@@ -19,12 +22,26 @@ public partial class IndexReadOperation
 
         foreach (var field in query.Metadata.OrderBy)
         {
-            // https://issues.hibernatingrhinos.com/issue/RavenDB-17888
-            DevelopmentHelper.ShardingToDo(DevelopmentHelper.TeamMember.Grisha, DevelopmentHelper.Severity.Normal,
-                "make it work without storing the fields in the index");
+            string fieldName;
+            object fieldValue;
```

Comment:
I wonder if we can avoid the boxing here.

Reply:
done